etsin_finder/frontend/js/components/qvain/index.jsx

```diff
@@ -34,6 +34,15 @@ class Qvain extends Component {
     this.props.Stores.Qvain.original = undefined
   }
 
+  IsJsonString = (str) => {
+    try {
+        JSON.parse(str);
```

Comment:
This is fine, but a better solution would be something like [this (StackOverflow question on checking if a string is valid JSON)](https://stackoverflow.com/questions/3710204/how-to-check-if-a-string-is-a-valid-json-string-in-javascript-without-using-try)

Reply:
Also the function is capitalized (should be camel-case)

Reply:
Ok I was wrong, JSON.parse is the better way to do it.